src/sentry/lint/engine.py

```diff
@@ -213,6 +220,46 @@ def js_test(file_list=None):
     return has_errors
 
 
+def less_format(file_list=None):
+    """
+    We only format less code as part of this pre-commit hook. It is not part
+    of the lint engine.
+    """
+    project_root = os.path.join(os.path.dirname(__file__), os.pardir, os.pardir, os.pardir)
+    prettier_path = os.path.join(project_root, 'node_modules', '.bin', 'prettier')
```

Comment:
maybe some of these variables should be lifted up for DRYness but just a nit

Reply:
afaik it doesn't look like `PRETTIER_VERSION` was ever used, the out of date message checks the version you have installed in `node_modules` vs the version defined in `package.json`